core/lib/configuration.ts

```diff
@@ -763,8 +764,7 @@ export class Configuration {
           // @<org>/<pkg>@<version>
           // <path>
           // if the entry starts with an @ it's definitely a package reference
-
-          if (await isDirectory(useEntry)) {
+          if (useEntry.endsWith('.tgz') || await isDirectory(useEntry)) {
```

Comment:
Any value in making the `endsWith` check a function like `isPackage` since it occurs more than once?

Reply:
Meh. I dunno--they're not actually checking for the same purpose -- this one passes the path thru to the package manager as the 'location/version', the other one is making sure we don't try to physically load it from the path.  The implementation could easily diverge in the future.

Reply:
Sounds good!